chat-plugins/info.js

```diff
@@ -1482,14 +1482,16 @@ exports.commands = {
 			}
 			let formatType = (format.gameType || "singles");
 			formatType = formatType.charAt(0).toUpperCase() + formatType.slice(1).toLowerCase();
-			if (!format.desc) {
+			if (!format.desc && !format.threads) {
```

Comment:
Why did we split `.desc` into `.desc` and `.threads`, anyway?

Reply:
Without the split, you get this error in <code>globals.ts:</code> <code>Interface 'Format' cannot simultaneously extend types 'Effect' and 'FormatsData'.
  Named property 'desc' of types 'Effect' and 'FormatsData' are not identical.</code>

Reply:
Oh, makes sense.